osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -148,11 +149,21 @@ protected override void LogoArriving(OsuLogo logo, bool resuming)
         {
             base.LogoArriving(logo, resuming);
 
-            logo.ScaleTo(new Vector2(0.15f), 300, Easing.In);
-            logo.MoveTo(new Vector2(0.5f), 300, Easing.In);
+            const double duration = 300;
+
+            logo.ScaleTo(new Vector2(0.15f), duration, Easing.In);
+            logo.MoveTo(new Vector2(0.5f), duration, Easing.In);
             logo.FadeIn(350);
 
-            logo.Delay(resuming ? 0 : 500).MoveToOffset(new Vector2(0, -0.24f), 500, Easing.InOutExpo);
+            facadeContainer.SetLogo(logo, 0.3f, 500, Easing.InOutQuint);
```

Comment:
where does the `0.3` come from?

Reply:
also was the change from expo to quint intentional? and the removal of the delay?

Reply:
unintentional, will add back

Reply:
I actually gave it an arbitrary scale since I didn't know how much padding it should be getting considering it was just a constant move before, open to suggestions

Reply:
Match the previous positioning for now.

Reply:
![Screen Shot 2019-03-27 at 8 20 45 PM](https://user-images.githubusercontent.com/5578000/55072724-32aa8880-50cf-11e9-84d1-beeb65be3598.png)

This is the size I'm using since it was what the extracted logic used inside ButtonSystem, but maybe this sizing is too specific for button system. If I try to just use 1x size, it's too big.

Reply:
I've resolved this by properly accounting for the `OsuLogo` scale in the facade, which seems to have gotten rid of the need for a magic number inside `SizeForFlow` (it's basically applying another scale down on top of `facadeScale`). It looks to be correct visually both on menu and in `PlayerLoader`, but could still use some further scrutinizing.